<ui version="4.0" >
 <class>tmsetting</class>
 <widget class="QWidget" name="tmsetting" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Translation Memory Settings</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QTabWidget" name="tabOptions" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>&amp;File</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="3" column="1" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Locations:</string>
         </property>
        </widget>
       </item>
       <item rowspan="3" row="1" column="0" >
        <widget class="QListView" name="listView" />
       </item>
       <item row="4" column="0" >
        <widget class="QCheckBox" name="checkBox" >
         <property name="text" >
          <string>Dive into Subfolders</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QPushButton" name="btnRemove" >
         <property name="text" >
          <string>&amp;Remove</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QPushButton" name="btnAdd" >
         <property name="text" >
          <string>&amp;Add</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>&amp;Database</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_4" >
      <attribute name="title" >
       <string>Options</string>
      </attribute>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Maximum string length</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QSpinBox" name="spinMaxLen" >
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="value" >
          <number>70</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QCheckBox" name="chkIgnorFuzzy" >
         <property name="text" >
          <string> Ignor fuzzy strings</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QSpinBox" name="spinSimilarity" >
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="singleStep" >
          <number>1</number>
         </property>
         <property name="value" >
          <number>75</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Similarity</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QCheckBox" name="chkCaseSensitive" >
         <property name="text" >
          <string>Case Sensitive</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QSpinBox" name="spinMaxCandidate" >
         <property name="maximum" >
          <number>100</number>
         </property>
         <property name="minimum" >
          <number>1</number>
         </property>
         <property name="value" >
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Maximum candidates</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="btnOk" >
     <property name="text" >
      <string>&amp;OK</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
